tests/tasks/setup/eks/awscli-mng.yaml

```diff
@@ -19,14 +19,23 @@ spec:
     description: The desired number of nodes in the cluster.
   - name: min-nodes
     default: "1"
-    description: The minimum number of nodes in the cluster.
+    description: The minimum number of nodes in the cluster nodegroup.
   - name: max-nodes
     default: "1000"
-    description: The maximum number of nodes in the cluster.
+    description: The maximum number of nodes in the cluster nodegroup.
   - name: endpoint
     default: ""
-  - name: host-cluster-node-role-arn
-    description: arn of the hostcluster node role. This tightly coupled to code here  - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/3ed1bbd47f7b8f111208e977acaa3edfa1834ca8/substrate/pkg/controller/substrate/cluster/addons/karpenter.go#L52 so if it's changed there, it should be changed here. This helps us to avoid creating a separate noderole for nodegroups.
+  - name: host-cluster-node-role-name
+    description: name of the hostcluster node role. This tightly coupled to code here  - https://github.com/awslabs/kubernetes-iteration-toolkit/blob/3ed1bbd47f7b8f111208e977acaa3edfa1834ca8/substrate/pkg/controller/substrate/cluster/addons/karpenter.go#L52 so if it's changed there, it should be changed here. This helps us to avoid creating a separate noderole for nodegroups.
```

Comment:
This description may not be valid anymore especially for `EKS Managed cluster` ?

Reply:
updated.